src/core/configuration/react/element.ts

```diff
@@ -1,4 +1,4 @@
-import { ITemplateMeta, Template } from '../../template';
+import { ITemplateMeta, Template as TemplateFunc } from '../../template';
```

Comment:
Bad naming. Template is not a function. It is a Pure Component.
Better `TemplateComponent`  or `TemplateComp`